{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "642944bc",
   "metadata": {},
   "outputs": [],
   "source": [
    "import GEOparse\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "18aef828",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load GEO Series\n",
    "gse = GEOparse.get_GEO(geo=\"GSE183947\", destdir=\"GEO\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "4e9e97db",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': ['Identification of five cytotoxicity-related genes involved in the progression of breast cancer'], 'geo_accession': ['GSE183947'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Jan 31 2022'], 'pubmed_id': ['35046993'], 'summary': ['Breast cancer is one of the deadly tumors in women, and its incidence continues to increase. This study aimed to identify novel therapeutic molecules using RNA sequencing (RNAseq) data of breast cancer from our hospitals.'], 'overall_design': ['30 pairs of normal and cancerous tissues from the same excision were collected from the Affiliated Cancer Hospital of Guangzhou Medical University, the Affiliated Cancer Hospital of Sun Yat-sen University and Guangzhou Army General Hospital. RNA sequencing was performed by Guangzhou Huayin Health medical Group. Original reads of RNA sequencing data were normalized as FPKM data.'], 'type': ['Expression profiling by high throughput sequencing'], 'contributor': ['Yan,,Zhang', 'Xu,X,Wei', 'Wei,K,Yang', 'Hong,,Shen'], 'sample_id': ['GSM5574685', 'GSM5574686', 'GSM5574687', 'GSM5574688', 'GSM5574689', 'GSM5574690', 'GSM5574691', 'GSM5574692', 'GSM5574693', 'GSM5574694', 'GSM5574695', 'GSM5574696', 'GSM5574697', 'GSM5574698', 'GSM5574699', 'GSM5574700', 'GSM5574701', 'GSM5574702', 'GSM5574703', 'GSM5574704', 'GSM5574705', 'GSM5574706', 'GSM5574707', 'GSM5574708', 'GSM5574709', 'GSM5574710', 'GSM5574711', 'GSM5574712', 'GSM5574713', 'GSM5574714', 'GSM5574715', 'GSM5574716', 'GSM5574717', 'GSM5574718', 'GSM5574719', 'GSM5574720', 'GSM5574721', 'GSM5574722', 'GSM5574723', 'GSM5574724', 'GSM5574725', 'GSM5574726', 'GSM5574727', 'GSM5574728', 'GSM5574729', 'GSM5574730', 'GSM5574731', 'GSM5574732', 'GSM5574733', 'GSM5574734', 'GSM5574735', 'GSM5574736', 'GSM5574737', 'GSM5574738', 'GSM5574739', 'GSM5574740', 'GSM5574741', 'GSM5574742', 'GSM5574743', 'GSM5574744'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183947/suppl/GSE183947_fpkm.csv.gz'], 'platform_id': ['GPL11154'], 'platform_taxid': ['9606'], 'sample_taxid': ['9606'], 'relation': ['BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA762469', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRP336638']}\n"
     ]
    }
   ],
   "source": [
    "# Explore metadata\n",
    "print(gse.metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "7cedcb4f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': ['tumor rep1'], 'geo_accession': ['GSM5574685'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 102548'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.102548'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395376', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143676'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep2'], 'geo_accession': ['GSM5574686'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 104338'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.104338'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395377', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143617'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep3'], 'geo_accession': ['GSM5574687'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 105094'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.105094'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395378', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143618'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep4'], 'geo_accession': ['GSM5574688'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 109745'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.109745'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395379', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143619'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep5'], 'geo_accession': ['GSM5574689'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 1906415'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1906415'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394912', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143620'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep6'], 'geo_accession': ['GSM5574690'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 1912627'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1912627'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394913', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143621'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep7'], 'geo_accession': ['GSM5574691'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 1924346'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1924346'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394914', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143622'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep8'], 'geo_accession': ['GSM5574692'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 1926760'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1926760'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394915', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143623'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep9'], 'geo_accession': ['GSM5574693'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 1927842'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1927842'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394916', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143624'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep10'], 'geo_accession': ['GSM5574694'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 1933414'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1933414'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394917', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143625'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep11'], 'geo_accession': ['GSM5574695'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 1940640'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.1940640'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394918', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143626'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep12'], 'geo_accession': ['GSM5574696'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2004407'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2004407'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394919', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143627'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep13'], 'geo_accession': ['GSM5574697'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2005288'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2005288'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394920', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143628'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep14'], 'geo_accession': ['GSM5574698'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2006047'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2006047'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394921', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143629'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep15'], 'geo_accession': ['GSM5574699'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2008260'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2008260'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394922', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143630'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep16'], 'geo_accession': ['GSM5574700'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2009329'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2009329'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394923', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143631'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep17'], 'geo_accession': ['GSM5574701'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2009381'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2009381'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394924', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143632'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep18'], 'geo_accession': ['GSM5574702'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2009850'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2009850'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394925', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143633'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep19'], 'geo_accession': ['GSM5574703'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2017611'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2017611'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394926', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143634'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep20'], 'geo_accession': ['GSM5574704'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2039179'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2039179'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394927', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143635'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep21'], 'geo_accession': ['GSM5574705'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2040686'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2040686'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394928', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143636'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep22'], 'geo_accession': ['GSM5574706'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 2045012'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2045012'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394929', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143637'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep23'], 'geo_accession': ['GSM5574707'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 2046297'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.2046297'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394930', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143638'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep24'], 'geo_accession': ['GSM5574708'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 348981'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.348981'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394931', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143639'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep25'], 'geo_accession': ['GSM5574709'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 354300'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.354300'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394932', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143640'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep26'], 'geo_accession': ['GSM5574710'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 359448'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.359448'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394933', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143641'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep27'], 'geo_accession': ['GSM5574711'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 94377'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.94377'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394934', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143642'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep28'], 'geo_accession': ['GSM5574712'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 98389'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.98389'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394935', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143643'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep29'], 'geo_accession': ['GSM5574713'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: no', 'donor: 98475'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.98475'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394936', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143644'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['tumor rep30'], 'geo_accession': ['GSM5574714'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: breast tumor', 'metastasis: yes', 'donor: 99145'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CA.99145'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394937', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143645'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep1'], 'geo_accession': ['GSM5574715'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 102548'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.102548'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394938', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143646'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep2'], 'geo_accession': ['GSM5574716'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 104338'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.104338'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394939', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143647'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep3'], 'geo_accession': ['GSM5574717'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 105094'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.105094'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394940', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143648'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep4'], 'geo_accession': ['GSM5574718'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 109745'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.109745'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394941', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143649'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep5'], 'geo_accession': ['GSM5574719'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 1906415'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1906415'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394942', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143650'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep6'], 'geo_accession': ['GSM5574720'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 1912627'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1912627'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394943', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143651'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep7'], 'geo_accession': ['GSM5574721'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 1924346'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1924346'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394944', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143652'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep8'], 'geo_accession': ['GSM5574722'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 1926760'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1926760'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394945', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143653'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep9'], 'geo_accession': ['GSM5574723'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 1927842'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1927842'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394946', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143654'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep10'], 'geo_accession': ['GSM5574724'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 1933414'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1933414'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394947', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143655'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep11'], 'geo_accession': ['GSM5574725'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 1940640'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.1940640'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394948', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143656'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep12'], 'geo_accession': ['GSM5574726'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2004407'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2004407'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394949', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143657'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep13'], 'geo_accession': ['GSM5574727'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2005288'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2005288'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394950', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143658'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep14'], 'geo_accession': ['GSM5574728'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2006047'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2006047'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394951', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143659'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep15'], 'geo_accession': ['GSM5574729'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2008260'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2008260'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394952', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143660'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep16'], 'geo_accession': ['GSM5574730'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2009329'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2009329'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394953', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143661'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep17'], 'geo_accession': ['GSM5574731'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2009381'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2009381'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394954', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143662'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep18'], 'geo_accession': ['GSM5574732'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2009850'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2009850'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21394955', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143663'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep19'], 'geo_accession': ['GSM5574733'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2017611'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2017611'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395380', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143664'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep20'], 'geo_accession': ['GSM5574734'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2039179'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2039179'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395381', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143665'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep21'], 'geo_accession': ['GSM5574735'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2040686'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2040686'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395382', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143666'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep22'], 'geo_accession': ['GSM5574736'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 2046297'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2046297'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395383', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143667'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep23'], 'geo_accession': ['GSM5574737'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 2046641'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.2046641'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395384', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143668'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep24'], 'geo_accession': ['GSM5574738'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 348981'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.348981'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395385', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143669'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep25'], 'geo_accession': ['GSM5574739'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 354300'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.354300'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395386', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143670'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep26'], 'geo_accession': ['GSM5574740'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 359448'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.359448'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395387', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143671'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep27'], 'geo_accession': ['GSM5574741'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 94377'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.94377'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395388', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143672'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep28'], 'geo_accession': ['GSM5574742'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 98389'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.98389'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395389', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143673'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep29'], 'geo_accession': ['GSM5574743'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: no', 'donor: 98475'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.98475'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395390', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143674'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n",
      "{'title': ['normal rep30'], 'geo_accession': ['GSM5574744'], 'status': ['Public on Sep 15 2021'], 'submission_date': ['Sep 11 2021'], 'last_update_date': ['Sep 15 2021'], 'type': ['SRA'], 'channel_count': ['1'], 'source_name_ch1': ['breast'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['tissue: normal breast tissue', 'metastasis: yes', 'donor: 99145'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Total RNA was isolated and purified using\\xa0TRIzol (Life, cat.265709, CA, USA) following the manufacturer's procedure.\", 'After the quality inspection of Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and NanoPhotometer (Implen, cat.N60, Munich, Germany), mRNA\\xa0with poly(A)\\xa0is purified from 1μg total RNA using VAHTS mRNA Capture Beads with Oligo (dT) (Vazyme, cat.N401-01, Nanjing, China) through\\xa0two rounds of purification. Subsequently, mRNA fragment was interrupted using VAHTS Universal V6 RNA-seq Library Prep Kit (Vazyme, cat.NR604, Nanjing, China) under 94℃ 8min\\xa0and reversed transcription into cDNA which would use to synthesise U-labeled second-stranded DNAs.\\xa0An A-base was added to the blunt ends of each strand\\xa0to ligase the indexed adapters\\xa0which contains a T-base at the tail end.\\xa0After UDG enzyme treatment of the U-labeled double-strand DNA, size selection was performed with VAHTS DNA Clean Beads (Vazyme, cat.N411, Nanjing, China). Then the ligated products are amplified with PCR by the following conditions: initial denaturation at 98℃ for 5 min; 12-17 cycles of denaturation at 98℃ for 10 sec, annealing at 60℃ for 30 sec, and extension at 72℃ for 30 sec; final extension at 72℃ for 5 min. The average insert size of\\xa0cDNA library was 280±80 bp. After purification by VAHTS DNA Clean Beads (Vazyme, cat.N411-02, Nanjing, China), quality control of concentration and fragment size is performed by Agilent 2100 Bioanalyzer (Agilent, cat.G2939AA, CA, USA) and Qubit assay tubes (Life, cat. 1604220, CA, USA).'], 'description': ['CAP.99145'], 'data_processing': ['In order to remove technical sequences, including adapters, polymerase chain reaction (PCR) primers, or fragments thereof, and quality of bases lower than 20, pass filter data of fastq format were processed by Cutadapt (V1.9.1) to be high quality clean data.', 'Firstly, reference genome sequences and gene model annotation files of relative species were downloaded from genome ENSEMBL website, Tophat2 (v2.1.0) was used to index reference genome sequence. Finally, clean data were aligned to reference genome via software Tophat2 (v2.1.02).', 'In the beginning transcripts in fasta format are converted from known gff annotation file and indexed properly. Then, with the file as a reference gene file, RSEM (v1.3.1) estimated gene and isoform expression levels from the pair-end clean data.', 'Genome_build: Homo sapiens (GRCh37)', 'Supplementary_files_format_and_content: fpkm.csv: Comma-separated text file includes FPKM values for each Sample.'], 'platform_id': ['GPL11154'], 'contact_name': ['Jianhua,,Xu'], 'contact_institute': ['Department of Laboratory Medicine Shunde Hospital of Guangzhou University of Chinese Medicine'], 'contact_address': ['No. 12, Jinsha Avenue, Shunde District, Foshan, Guangdong, P.R. China'], 'contact_city': ['Fushan'], 'contact_state': ['Guangdong'], 'contact_zip/postal_code': ['528300'], 'contact_country': ['China'], 'instrument_model': ['Illumina HiSeq 2000'], 'library_selection': ['cDNA'], 'library_source': ['transcriptomic'], 'library_strategy': ['RNA-Seq'], 'relation': ['BioSample: https://www.ncbi.nlm.nih.gov/biosample/SAMN21395391', 'SRA: https://www.ncbi.nlm.nih.gov/sra?term=SRX12143675'], 'supplementary_file_1': ['NONE'], 'series_id': ['GSE183947'], 'data_row_count': ['0']}\n"
     ]
    }
   ],
   "source": [
    "# Access sample data\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    print(gsm.metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "d8e6a0ce",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "60\n",
      "dict_keys(['title', 'geo_accession', 'status', 'submission_date', 'last_update_date', 'pubmed_id', 'summary', 'overall_design', 'type', 'contributor', 'sample_id', 'contact_name', 'contact_institute', 'contact_address', 'contact_city', 'contact_state', 'contact_zip/postal_code', 'contact_country', 'supplementary_file', 'platform_id', 'platform_taxid', 'sample_taxid', 'relation'])\n",
      "['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183947/suppl/GSE183947_fpkm.csv.gz']\n"
     ]
    }
   ],
   "source": [
    "print(len(gse.gsms))  # should be 60\n",
    "print(gse.metadata.keys())\n",
    "print(gse.metadata.get(\"supplementary_file\"))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "33d5e306",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "✅ Expression data loaded successfully!\n",
      "Shape: (20246, 60)\n",
      "          CA.102548  CA.104338  CA.105094  CA.109745  CA.1906415  CA.1912627  \\\n",
      "TSPAN6         0.93       1.97       0.00       5.45        4.52        4.75   \n",
      "TNMD           0.00       0.00       0.00       0.00        0.00        0.00   \n",
      "DPM1           0.00       0.43       0.00       3.43        8.45        8.53   \n",
      "SCYL3          5.78       5.17       8.76       4.58        7.20        6.03   \n",
      "C1orf112       2.83       6.26       3.37       6.24        5.16       13.69   \n",
      "\n",
      "          CA.1924346  CA.1926760  CA.1927842  CA.1933414  ...  CAP.2040686  \\\n",
      "TSPAN6          3.96        3.58        6.41       11.89  ...         6.66   \n",
      "TNMD            0.00        0.23        0.39        0.44  ...         0.12   \n",
      "DPM1            7.80        7.62        6.40        6.09  ...         4.93   \n",
      "SCYL3           9.05        5.37        5.92       12.45  ...         8.02   \n",
      "C1orf112        6.69        5.28        7.65       13.71  ...         7.91   \n",
      "\n",
      "          CAP.2046297  CAP.2046641  CAP.348981  CAP.354300  CAP.359448  \\\n",
      "TSPAN6           8.35         8.94        6.33        5.94        6.35   \n",
      "TNMD             0.17         1.08        0.29        0.00        0.07   \n",
      "DPM1             7.47         5.72        4.96        9.28        9.15   \n",
      "SCYL3            6.00         5.28        4.98        4.45        7.00   \n",
      "C1orf112         4.61         8.35        9.84        7.68        5.62   \n",
      "\n",
      "          CAP.94377  CAP.98389  CAP.98475  CAP.99145  \n",
      "TSPAN6         3.74       4.84      10.46       4.54  \n",
      "TNMD           9.19       1.18       0.09       0.39  \n",
      "DPM1           4.77       3.75       7.31       2.77  \n",
      "SCYL3          4.14       5.51       7.45       2.33  \n",
      "C1orf112       2.81       7.08       7.28       5.39  \n",
      "\n",
      "[5 rows x 60 columns]\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import requests\n",
    "import gzip\n",
    "from io import BytesIO\n",
    "\n",
    "# Correct URL from metadata\n",
    "url = \"https://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183947/suppl/GSE183947_fpkm.csv.gz\"\n",
    "\n",
    "# Download and decompress\n",
    "r = requests.get(url)\n",
    "\n",
    "# Load gzipped CSV\n",
    "df = pd.read_csv(BytesIO(gzip.decompress(r.content)), index_col=0)\n",
    "\n",
    "print(\"✅ Expression data loaded successfully!\")\n",
    "print(\"Shape:\", df.shape)\n",
    "print(df.head())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "616c659b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "               CA.102548  CA.104338  CA.105094  CA.109745  CA.1906415  \\\n",
      "TSPAN6              0.93       1.97       0.00       5.45        4.52   \n",
      "TNMD                0.00       0.00       0.00       0.00        0.00   \n",
      "DPM1                0.00       0.43       0.00       3.43        8.45   \n",
      "SCYL3               5.78       5.17       8.76       4.58        7.20   \n",
      "C1orf112            2.83       6.26       3.37       6.24        5.16   \n",
      "...                  ...        ...        ...        ...         ...   \n",
      "RP11-1084J3.4       0.00       0.16       0.00       0.00        0.00   \n",
      "RP11-944L7.5        0.00       0.00       0.00       0.00        0.00   \n",
      "FLJ00388            0.00       0.00       0.00       0.83        0.00   \n",
      "RP11-474G23.1       0.00       0.00       0.00       0.10        0.64   \n",
      "AC005358.1          0.00       0.00       0.00       0.00        0.00   \n",
      "\n",
      "               CA.1912627  CA.1924346  CA.1926760  CA.1927842  CA.1933414  \\\n",
      "TSPAN6               4.75        3.96        3.58        6.41       11.89   \n",
      "TNMD                 0.00        0.00        0.23        0.39        0.44   \n",
      "DPM1                 8.53        7.80        7.62        6.40        6.09   \n",
      "SCYL3                6.03        9.05        5.37        5.92       12.45   \n",
      "C1orf112            13.69        6.69        5.28        7.65       13.71   \n",
      "...                   ...         ...         ...         ...         ...   \n",
      "RP11-1084J3.4        0.30        0.00        0.00        0.43        0.55   \n",
      "RP11-944L7.5         0.00        0.00        0.00        0.00        0.00   \n",
      "FLJ00388             0.00        0.00        0.00        0.00        0.00   \n",
      "RP11-474G23.1        0.00        0.00        0.00        0.20        0.27   \n",
      "AC005358.1           0.00        0.00        0.06        0.37        0.00   \n",
      "\n",
      "               ...  CAP.2040686  CAP.2046297  CAP.2046641  CAP.348981  \\\n",
      "TSPAN6         ...         6.66         8.35         8.94        6.33   \n",
      "TNMD           ...         0.12         0.17         1.08        0.29   \n",
      "DPM1           ...         4.93         7.47         5.72        4.96   \n",
      "SCYL3          ...         8.02         6.00         5.28        4.98   \n",
      "C1orf112       ...         7.91         4.61         8.35        9.84   \n",
      "...            ...          ...          ...          ...         ...   \n",
      "RP11-1084J3.4  ...         0.19         0.72         0.95        0.00   \n",
      "RP11-944L7.5   ...         0.00         0.00         0.00        0.00   \n",
      "FLJ00388       ...         0.00         0.00         0.00        0.00   \n",
      "RP11-474G23.1  ...         0.46         0.19         0.70        0.16   \n",
      "AC005358.1     ...         0.00         2.85         0.46        0.00   \n",
      "\n",
      "               CAP.354300  CAP.359448  CAP.94377  CAP.98389  CAP.98475  \\\n",
      "TSPAN6               5.94        6.35       3.74       4.84      10.46   \n",
      "TNMD                 0.00        0.07       9.19       1.18       0.09   \n",
      "DPM1                 9.28        9.15       4.77       3.75       7.31   \n",
      "SCYL3                4.45        7.00       4.14       5.51       7.45   \n",
      "C1orf112             7.68        5.62       2.81       7.08       7.28   \n",
      "...                   ...         ...        ...        ...        ...   \n",
      "RP11-1084J3.4        0.33        0.26       0.00       0.00       0.00   \n",
      "RP11-944L7.5         0.00        0.00       0.00       0.00       0.00   \n",
      "FLJ00388             0.00        0.00       4.37       0.00       0.00   \n",
      "RP11-474G23.1        0.84        0.58       0.00       0.00       0.00   \n",
      "AC005358.1           0.00        0.42       0.62       0.00       3.31   \n",
      "\n",
      "               CAP.99145  \n",
      "TSPAN6              4.54  \n",
      "TNMD                0.39  \n",
      "DPM1                2.77  \n",
      "SCYL3               2.33  \n",
      "C1orf112            5.39  \n",
      "...                  ...  \n",
      "RP11-1084J3.4       0.23  \n",
      "RP11-944L7.5        0.00  \n",
      "FLJ00388            0.00  \n",
      "RP11-474G23.1       0.14  \n",
      "AC005358.1          1.19  \n",
      "\n",
      "[20246 rows x 60 columns]\n"
     ]
    }
   ],
   "source": [
    "print(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "c61157a7",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "dict_keys(['title', 'geo_accession', 'status', 'submission_date', 'last_update_date', 'type', 'channel_count', 'source_name_ch1', 'organism_ch1', 'taxid_ch1', 'characteristics_ch1', 'molecule_ch1', 'extract_protocol_ch1', 'description', 'data_processing', 'platform_id', 'contact_name', 'contact_institute', 'contact_address', 'contact_city', 'contact_state', 'contact_zip/postal_code', 'contact_country', 'instrument_model', 'library_selection', 'library_source', 'library_strategy', 'relation', 'supplementary_file_1', 'series_id', 'data_row_count'])\n",
      "['GPL11154']\n"
     ]
    }
   ],
   "source": [
    "gsm = list(gse.gsms.values())[0]\n",
    "print(gsm.metadata.keys())\n",
    "print(gsm.metadata.get(\"platform_id\"))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "300779b1",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "✅ Phenotype data shape: (60, 5)\n",
      "          GSM        title        tissue metastasis    donor\n",
      "0  GSM5574685   tumor rep1  breast tumor        yes   102548\n",
      "1  GSM5574686   tumor rep2  breast tumor        yes   104338\n",
      "2  GSM5574687   tumor rep3  breast tumor        yes   105094\n",
      "3  GSM5574688   tumor rep4  breast tumor         no   109745\n",
      "4  GSM5574689   tumor rep5  breast tumor         no  1906415\n",
      "5  GSM5574690   tumor rep6  breast tumor        yes  1912627\n",
      "6  GSM5574691   tumor rep7  breast tumor         no  1924346\n",
      "7  GSM5574692   tumor rep8  breast tumor         no  1926760\n",
      "8  GSM5574693   tumor rep9  breast tumor        yes  1927842\n",
      "9  GSM5574694  tumor rep10  breast tumor         no  1933414\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "meta = []\n",
    "\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    characteristics = gsm.metadata.get(\"characteristics_ch1\", [])\n",
    "    \n",
    "    # Convert key:value pairs into dictionary entries\n",
    "    char_dict = {}\n",
    "    for item in characteristics:\n",
    "        if \":\" in item:\n",
    "            key, val = item.split(\":\", 1)\n",
    "            char_dict[key.strip()] = val.strip()\n",
    "    \n",
    "    meta.append({\n",
    "        \"GSM\": gsm_name,\n",
    "        \"title\": gsm.metadata.get(\"title\", [\"\"])[0],\n",
    "        **char_dict  # unpack characteristics\n",
    "    })\n",
    "\n",
    "pheno = pd.DataFrame(meta)\n",
    "print(\"✅ Phenotype data shape:\", pheno.shape)\n",
    "print(pheno.head(10))\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "myytcode",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
