{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "642944bc",
   "metadata": {},
   "outputs": [],
   "source": [
    "import GEOparse\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "18aef828",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Load GEO Series\n",
    "# Download and parse (destdir will store files in ../GEO)\n",
    "gse = GEOparse.get_GEO(\"GSE197406\", destdir=\"../GEO\", how=\"full\")\n",
    "print(\"Loaded GSE with\", len(gse.gsms), \"GSM samples\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "id": "baf5db7c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'GSM5916425': <SAMPLE: GSM5916425>, 'GSM5916426': <SAMPLE: GSM5916426>, 'GSM5916427': <SAMPLE: GSM5916427>, 'GSM5916428': <SAMPLE: GSM5916428>, 'GSM5916429': <SAMPLE: GSM5916429>, 'GSM5916430': <SAMPLE: GSM5916430>, 'GSM5916431': <SAMPLE: GSM5916431>, 'GSM5916432': <SAMPLE: GSM5916432>, 'GSM5916433': <SAMPLE: GSM5916433>, 'GSM5916434': <SAMPLE: GSM5916434>, 'GSM5916435': <SAMPLE: GSM5916435>, 'GSM5916436': <SAMPLE: GSM5916436>, 'GSM5916437': <SAMPLE: GSM5916437>, 'GSM5916438': <SAMPLE: GSM5916438>, 'GSM5916439': <SAMPLE: GSM5916439>}\n"
     ]
    }
   ],
   "source": [
    "print(gse.gsms)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 65,
   "id": "cc4f1f50",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Columns in GSM table: ['ID_REF', 'VALUE']\n",
      "        ID_REF     VALUE\n",
      "0    217771_at   6.37042\n",
      "1    206606_at  11.61950\n",
      "2  215712_s_at   9.16470\n",
      "3   1559573_at  10.70350\n",
      "4  209978_s_at  14.51450\n"
     ]
    }
   ],
   "source": [
    "example = gse.gsms[gsm_keys[0]]\n",
    "print(\"Columns in GSM table:\", example.table.columns.tolist())\n",
    "# optionally preview first 5 rows:\n",
    "print(example.table.head())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "4e9e97db",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': ['Expression data from Wilson disease patients liver'], 'geo_accession': ['GSE197406'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 23 2025'], 'pubmed_id': ['40140071'], 'summary': ['The goal of the study was to analyze gene expression profle from the WD patients who underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.', 'We used microarrays (Affymetrix) to determine the gene expression profile in WD'], 'overall_design': ['Liver sections were snap frozen in liquid nitrogen and stored. Total RNA was isolated using TRIZOL reagent (Invitrogen, Grand Island, NY) followed by RNeasy cleanup procedure (Qiagen, Valencia, CA). The integrity of isolated RNA was electrophoretically verified by ethidium bromide staining and by optical densities (OD) ratio (OD260nm/280nm>1.8). A total of 15 liver RNA samples (8 biological replicates for control liver and 7 for WD patients) were examined for RNA integrity and concentration on an Agilent 2100 Bioanalyzer (Agilent Technologies, Palo Alto, CA, USA) using the RNA 6.000 LabChip Kit (Agilent Technologies) according to the manufacturers instructions. Affymetrix GeneChip analysis was conducted at the microarray core facility of the Interdisziplinäres Zentrum für klinische Forschung (IZKF), Leipzig (Faculty of Medicine, University of Leipzig). 2 μg of total RNA were used to prepare double-stranded cDNA (Superscript II, Life Technologies, Gaithersburg, MD) primed with oligo-dT containing an T7 RNA polymerase promoter site (Genset SA, Paris, France). cDNA was purified by phenol- chloroform extraction before in vitro transcription using the IVT labeling kit (Affymetrix, Santa Clara, CA, USA) to synthesize cRNA. After the in vitro transcription, unincorporated nucleotides were removed using the RNeasy kit (QIAGEN). The cRNA was fragmented and hybridized to two different (technical replicates) Human Genome U133 Plus 2.0 (Arrays Affymetrix). The washing and staining of the probe array was performed according to the manufacturer’s instructions. The array was scanned with a third generation Affymetrix GeneChipScanner 3000.'], 'type': ['Expression profiling by array'], 'contributor': ['S,,Dev', 'D,,Huster', 'S,,Lutsenko'], 'sample_id': ['GSM5916425', 'GSM5916426', 'GSM5916427', 'GSM5916428', 'GSM5916429', 'GSM5916430', 'GSM5916431', 'GSM5916432', 'GSM5916433', 'GSM5916434', 'GSM5916435', 'GSM5916436', 'GSM5916437', 'GSM5916438', 'GSM5916439'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_Human_WD_microarray_processed_file.xlsx', 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_Metadata_worksheet.xlsx', 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_RAW.tar'], 'platform_id': ['GPL570'], 'platform_taxid': ['9606'], 'sample_taxid': ['9606'], 'relation': ['BioProject: https://www.ncbi.nlm.nih.gov/bioproject/PRJNA810169']}\n"
     ]
    }
   ],
   "source": [
    "# Explore metadata\n",
    "print(gse.metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "c690c588",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "                        Key                                              Value\n",
      "0                     title  [Expression data from Wilson disease patients ...\n",
      "1             geo_accession                                        [GSE197406]\n",
      "2                    status                            [Public on May 22 2025]\n",
      "3           submission_date                                      [Feb 24 2022]\n",
      "4          last_update_date                                      [May 23 2025]\n",
      "5                 pubmed_id                                         [40140071]\n",
      "6                   summary  [The goal of the study was to analyze gene exp...\n",
      "7            overall_design  [Liver sections were snap frozen in liquid nit...\n",
      "8                      type                    [Expression profiling by array]\n",
      "9               contributor                   [S,,Dev, D,,Huster, S,,Lutsenko]\n",
      "10                sample_id  [GSM5916425, GSM5916426, GSM5916427, GSM591642...\n",
      "11             contact_name                                         [Som,,Dev]\n",
      "12       contact_laboratory                                    [Hunterian 207]\n",
      "13       contact_department                   [Physiology, School of Medicine]\n",
      "14        contact_institute                         [Johns Hopkins University]\n",
      "15          contact_address                                  [725 N, Wolfe St]\n",
      "16             contact_city                                        [Baltimore]\n",
      "17            contact_state                                               [MD]\n",
      "18  contact_zip/postal_code                                            [21205]\n",
      "19          contact_country                                              [USA]\n",
      "20       supplementary_file  [ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197n...\n",
      "21              platform_id                                           [GPL570]\n",
      "22           platform_taxid                                             [9606]\n",
      "23             sample_taxid                                             [9606]\n",
      "24                 relation  [BioProject: https://www.ncbi.nlm.nih.gov/biop...\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Convert metadata dictionary to DataFrame\n",
    "meta_df = pd.DataFrame(list(gse.metadata.items()), \n",
    "                       columns=[\"Key\", \"Value\"])\n",
    "\n",
    "# Display nicely\n",
    "print(meta_df)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "7cedcb4f",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'title': ['NL1'], 'geo_accession': ['GSM5916425'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Male', 'age: 61'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916425/suppl/GSM5916425_NL1_190506.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL2'], 'geo_accession': ['GSM5916426'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 58'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916426/suppl/GSM5916426_NL2_200406.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL3'], 'geo_accession': ['GSM5916427'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 45'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916427/suppl/GSM5916427_NL3_190506.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL4'], 'geo_accession': ['GSM5916428'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 46'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916428/suppl/GSM5916428_NL4_27.10.05.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL5'], 'geo_accession': ['GSM5916429'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Male', 'age: 52'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916429/suppl/GSM5916429_NL5_261005.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL6'], 'geo_accession': ['GSM5916430'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 56'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916430/suppl/GSM5916430_NL6_27.10.05.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL7'], 'geo_accession': ['GSM5916431'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 58'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916431/suppl/GSM5916431_NL7_200406.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['NL8'], 'geo_accession': ['GSM5916432'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human liver (no liver abnormality)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human liver (no liver abnormality)', 'tissue: Liver', 'genotype: Female', 'age: 62'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from Control liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916432/suppl/GSM5916432_NL8_200406.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD2'], 'geo_accession': ['GSM5916433'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Female', 'age: 38'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916433/suppl/GSM5916433_WND2_261005.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD3'], 'geo_accession': ['GSM5916434'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Male', 'age: 30'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916434/suppl/GSM5916434_WND3_27.10.05.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD4'], 'geo_accession': ['GSM5916435'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Female', 'age: 46'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916435/suppl/GSM5916435_WND4_221105.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD5'], 'geo_accession': ['GSM5916436'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Female', 'age: 19'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916436/suppl/GSM5916436_WND5_221105.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD6'], 'geo_accession': ['GSM5916437'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Female', 'age: 18'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916437/suppl/GSM5916437_WND6_190506.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD7'], 'geo_accession': ['GSM5916438'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Female', 'age: 20'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916438/suppl/GSM5916438_WND7_221105.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n",
      "{'title': ['WD8'], 'geo_accession': ['GSM5916439'], 'status': ['Public on May 22 2025'], 'submission_date': ['Feb 24 2022'], 'last_update_date': ['May 22 2025'], 'type': ['RNA'], 'channel_count': ['1'], 'source_name_ch1': ['Human WD liver (Cirrhosis)'], 'organism_ch1': ['Homo sapiens'], 'taxid_ch1': ['9606'], 'characteristics_ch1': ['disease state: Human WD liver (Cirrhosis)', 'tissue: Liver', 'genotype: Male', 'age: 39'], 'treatment_protocol_ch1': ['Liver sections were snap frozen in liquid nitrogen and stored, RNA was extracted'], 'growth_protocol_ch1': ['WD patients underwent liver transplantation for acute or chronic liver failure. Control specimens were obtained from patients who underwent liver resections for other clinical reasons.'], 'molecule_ch1': ['total RNA'], 'extract_protocol_ch1': [\"Trizol extraction of total RNA followed by Rneasy cleanup procedure performed according to the manufacturer's instructions.\"], 'label_ch1': ['IVT  labelling'], 'label_protocol_ch1': ['cRNA were prepared using IVT labelling kit (Affymetrixs) from cDNA and cDNA was synthesized from 2 ug total RNA (starting amount)'], 'hyb_protocol': ['Following fragmentation, 10 ug of cRNA were hybridized for 16 hr at 45C on GeneChip Drosophila Genome Array. GeneChips were washed and stained in the Affymetrix Fluidics Station 400.'], 'scan_protocol': ['GeneChips were scanned using third generation Affymetrixs Genechip Scanner 3000.'], 'description': ['Gene expression data from WD liver'], 'data_processing': ['The data were analyzed with Microarray Suite version 5.0 (MAS 5.0) using Affymetrix default analysis settings and ubjected to global scaling with an average target intensity of 350 to allow for direct comparison of hybridization values from different targets.'], 'platform_id': ['GPL570'], 'contact_name': ['Som,,Dev'], 'contact_laboratory': ['Hunterian 207'], 'contact_department': ['Physiology, School of Medicine'], 'contact_institute': ['Johns Hopkins University'], 'contact_address': ['725 N, Wolfe St'], 'contact_city': ['Baltimore'], 'contact_state': ['MD'], 'contact_zip/postal_code': ['21205'], 'contact_country': ['USA'], 'supplementary_file': ['ftp://ftp.ncbi.nlm.nih.gov/geo/samples/GSM5916nnn/GSM5916439/suppl/GSM5916439_WND8_190506.CEL.gz'], 'series_id': ['GSE197406'], 'data_row_count': ['54675']}\n"
     ]
    }
   ],
   "source": [
    "# Access sample data\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    print(gsm.metadata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "id": "206512f9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Key</th>\n",
       "      <th>Value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSM5916425</td>\n",
       "      <td>&lt;SAMPLE: GSM5916425&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSM5916426</td>\n",
       "      <td>&lt;SAMPLE: GSM5916426&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSM5916427</td>\n",
       "      <td>&lt;SAMPLE: GSM5916427&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSM5916428</td>\n",
       "      <td>&lt;SAMPLE: GSM5916428&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSM5916429</td>\n",
       "      <td>&lt;SAMPLE: GSM5916429&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>GSM5916430</td>\n",
       "      <td>&lt;SAMPLE: GSM5916430&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>GSM5916431</td>\n",
       "      <td>&lt;SAMPLE: GSM5916431&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>GSM5916432</td>\n",
       "      <td>&lt;SAMPLE: GSM5916432&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>GSM5916433</td>\n",
       "      <td>&lt;SAMPLE: GSM5916433&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>GSM5916434</td>\n",
       "      <td>&lt;SAMPLE: GSM5916434&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>GSM5916435</td>\n",
       "      <td>&lt;SAMPLE: GSM5916435&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>GSM5916436</td>\n",
       "      <td>&lt;SAMPLE: GSM5916436&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>GSM5916437</td>\n",
       "      <td>&lt;SAMPLE: GSM5916437&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>GSM5916438</td>\n",
       "      <td>&lt;SAMPLE: GSM5916438&gt;</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>GSM5916439</td>\n",
       "      <td>&lt;SAMPLE: GSM5916439&gt;</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           Key                 Value\n",
       "0   GSM5916425  <SAMPLE: GSM5916425>\n",
       "1   GSM5916426  <SAMPLE: GSM5916426>\n",
       "2   GSM5916427  <SAMPLE: GSM5916427>\n",
       "3   GSM5916428  <SAMPLE: GSM5916428>\n",
       "4   GSM5916429  <SAMPLE: GSM5916429>\n",
       "5   GSM5916430  <SAMPLE: GSM5916430>\n",
       "6   GSM5916431  <SAMPLE: GSM5916431>\n",
       "7   GSM5916432  <SAMPLE: GSM5916432>\n",
       "8   GSM5916433  <SAMPLE: GSM5916433>\n",
       "9   GSM5916434  <SAMPLE: GSM5916434>\n",
       "10  GSM5916435  <SAMPLE: GSM5916435>\n",
       "11  GSM5916436  <SAMPLE: GSM5916436>\n",
       "12  GSM5916437  <SAMPLE: GSM5916437>\n",
       "13  GSM5916438  <SAMPLE: GSM5916438>\n",
       "14  GSM5916439  <SAMPLE: GSM5916439>"
      ]
     },
     "execution_count": 46,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.DataFrame(list(gse.gsms.items()),\n",
    "             columns=[\"Key\" , \"Value\"])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "id": "d8e6a0ce",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "15\n",
      "dict_keys(['title', 'geo_accession', 'status', 'submission_date', 'last_update_date', 'pubmed_id', 'summary', 'overall_design', 'type', 'contributor', 'sample_id', 'contact_name', 'contact_laboratory', 'contact_department', 'contact_institute', 'contact_address', 'contact_city', 'contact_state', 'contact_zip/postal_code', 'contact_country', 'supplementary_file', 'platform_id', 'platform_taxid', 'sample_taxid', 'relation'])\n",
      "['ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_Human_WD_microarray_processed_file.xlsx', 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_Metadata_worksheet.xlsx', 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE197nnn/GSE197406/suppl/GSE197406_RAW.tar']\n"
     ]
    }
   ],
   "source": [
    "print(len(gse.gsms))  # should be 60\n",
    "print(gse.metadata.keys())\n",
    "print(gse.metadata.get(\"supplementary_file\"))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 58,
   "id": "33d5e306",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>GSM5916425</th>\n",
       "      <th>GSM5916426</th>\n",
       "      <th>GSM5916427</th>\n",
       "      <th>GSM5916428</th>\n",
       "      <th>GSM5916429</th>\n",
       "      <th>GSM5916430</th>\n",
       "      <th>GSM5916431</th>\n",
       "      <th>GSM5916432</th>\n",
       "      <th>GSM5916433</th>\n",
       "      <th>GSM5916434</th>\n",
       "      <th>GSM5916435</th>\n",
       "      <th>GSM5916436</th>\n",
       "      <th>GSM5916437</th>\n",
       "      <th>GSM5916438</th>\n",
       "      <th>GSM5916439</th>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>ID_REF</th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "      <th></th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1007_s_at</th>\n",
       "      <td>8.25153</td>\n",
       "      <td>7.76315</td>\n",
       "      <td>8.40313</td>\n",
       "      <td>7.98926</td>\n",
       "      <td>8.05217</td>\n",
       "      <td>8.44563</td>\n",
       "      <td>7.58667</td>\n",
       "      <td>8.02614</td>\n",
       "      <td>9.79436</td>\n",
       "      <td>9.66555</td>\n",
       "      <td>9.49759</td>\n",
       "      <td>9.89471</td>\n",
       "      <td>8.45370</td>\n",
       "      <td>8.85845</td>\n",
       "      <td>8.84316</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1053_at</th>\n",
       "      <td>6.74268</td>\n",
       "      <td>6.75010</td>\n",
       "      <td>6.50660</td>\n",
       "      <td>6.88312</td>\n",
       "      <td>6.97066</td>\n",
       "      <td>6.76910</td>\n",
       "      <td>6.89180</td>\n",
       "      <td>6.67075</td>\n",
       "      <td>6.93478</td>\n",
       "      <td>6.90036</td>\n",
       "      <td>6.62057</td>\n",
       "      <td>6.62056</td>\n",
       "      <td>6.52978</td>\n",
       "      <td>6.72519</td>\n",
       "      <td>6.55780</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117_at</th>\n",
       "      <td>6.74961</td>\n",
       "      <td>6.53426</td>\n",
       "      <td>6.72746</td>\n",
       "      <td>6.90891</td>\n",
       "      <td>6.80198</td>\n",
       "      <td>6.80546</td>\n",
       "      <td>6.76402</td>\n",
       "      <td>6.76341</td>\n",
       "      <td>6.74732</td>\n",
       "      <td>6.86814</td>\n",
       "      <td>6.90635</td>\n",
       "      <td>6.84800</td>\n",
       "      <td>6.63013</td>\n",
       "      <td>6.73454</td>\n",
       "      <td>6.55409</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121_at</th>\n",
       "      <td>8.79223</td>\n",
       "      <td>8.73107</td>\n",
       "      <td>9.32590</td>\n",
       "      <td>9.22579</td>\n",
       "      <td>9.28703</td>\n",
       "      <td>9.42747</td>\n",
       "      <td>9.04310</td>\n",
       "      <td>8.86239</td>\n",
       "      <td>9.51909</td>\n",
       "      <td>9.22565</td>\n",
       "      <td>9.39310</td>\n",
       "      <td>9.32185</td>\n",
       "      <td>9.42897</td>\n",
       "      <td>8.99391</td>\n",
       "      <td>8.99834</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1255_g_at</th>\n",
       "      <td>5.60172</td>\n",
       "      <td>5.69342</td>\n",
       "      <td>5.70023</td>\n",
       "      <td>5.77163</td>\n",
       "      <td>5.73856</td>\n",
       "      <td>5.75393</td>\n",
       "      <td>5.76626</td>\n",
       "      <td>5.64828</td>\n",
       "      <td>5.76027</td>\n",
       "      <td>5.73813</td>\n",
       "      <td>5.71184</td>\n",
       "      <td>5.73056</td>\n",
       "      <td>5.73223</td>\n",
       "      <td>5.73361</td>\n",
       "      <td>5.78678</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           GSM5916425  GSM5916426  GSM5916427  GSM5916428  GSM5916429  \\\n",
       "ID_REF                                                                  \n",
       "1007_s_at     8.25153     7.76315     8.40313     7.98926     8.05217   \n",
       "1053_at       6.74268     6.75010     6.50660     6.88312     6.97066   \n",
       "117_at        6.74961     6.53426     6.72746     6.90891     6.80198   \n",
       "121_at        8.79223     8.73107     9.32590     9.22579     9.28703   \n",
       "1255_g_at     5.60172     5.69342     5.70023     5.77163     5.73856   \n",
       "\n",
       "           GSM5916430  GSM5916431  GSM5916432  GSM5916433  GSM5916434  \\\n",
       "ID_REF                                                                  \n",
       "1007_s_at     8.44563     7.58667     8.02614     9.79436     9.66555   \n",
       "1053_at       6.76910     6.89180     6.67075     6.93478     6.90036   \n",
       "117_at        6.80546     6.76402     6.76341     6.74732     6.86814   \n",
       "121_at        9.42747     9.04310     8.86239     9.51909     9.22565   \n",
       "1255_g_at     5.75393     5.76626     5.64828     5.76027     5.73813   \n",
       "\n",
       "           GSM5916435  GSM5916436  GSM5916437  GSM5916438  GSM5916439  \n",
       "ID_REF                                                                 \n",
       "1007_s_at     9.49759     9.89471     8.45370     8.85845     8.84316  \n",
       "1053_at       6.62057     6.62056     6.52978     6.72519     6.55780  \n",
       "117_at        6.90635     6.84800     6.63013     6.73454     6.55409  \n",
       "121_at        9.39310     9.32185     9.42897     8.99391     8.99834  \n",
       "1255_g_at     5.71184     5.73056     5.73223     5.73361     5.78678  "
      ]
     },
     "execution_count": 58,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Extract expression data from all samples\n",
    "samples = []\n",
    "\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    df = gsm.table[[\"ID_REF\", \"VALUE\"]].copy()\n",
    "    df.columns = [\"ID_REF\", gsm_name]\n",
    "    samples.append(df)\n",
    "\n",
    "# Merge all sample expression columns by ID_REF\n",
    "import functools\n",
    "expr_df = functools.reduce(lambda left, right: pd.merge(left, right, on=\"ID_REF\", how=\"outer\"), samples)\n",
    "\n",
    "expr_df.set_index(\"ID_REF\", inplace=True)\n",
    "expr_df.head()\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "300779b1",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "07-Oct-2025 15:02:35 DEBUG utils - Directory ./data already exists. Skipping.\n",
      "07-Oct-2025 15:02:35 INFO GEOparse - File already exist: using local version.\n",
      "07-Oct-2025 15:02:35 INFO GEOparse - Parsing ./data/GSE183947_family.soft.gz: \n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - DATABASE: GeoMiame\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SERIES: GSE183947\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - PLATFORM: GPL11154\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574685\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574686\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574687\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574688\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574689\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574690\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574691\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574692\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574693\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574694\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574695\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574696\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574697\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574698\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574699\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574700\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574701\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574702\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574703\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574704\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574705\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574706\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574707\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574708\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574709\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574710\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574711\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574712\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574713\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574714\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574715\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574716\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574717\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574718\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574719\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574720\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574721\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574722\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574723\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574724\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574725\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574726\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574727\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574728\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574729\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574730\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574731\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574732\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574733\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574734\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574735\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574736\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574737\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574738\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574739\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574740\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574741\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574742\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574743\n",
      "07-Oct-2025 15:02:35 DEBUG GEOparse - SAMPLE: GSM5574744\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Pheno shape: (60, 2)\n",
      "          gsm        title\n",
      "0  GSM5574685   tumor rep1\n",
      "1  GSM5574686   tumor rep2\n",
      "2  GSM5574687   tumor rep3\n",
      "3  GSM5574688   tumor rep4\n",
      "4  GSM5574689   tumor rep5\n",
      "5  GSM5574690   tumor rep6\n",
      "6  GSM5574691   tumor rep7\n",
      "7  GSM5574692   tumor rep8\n",
      "8  GSM5574693   tumor rep9\n",
      "9  GSM5574694  tumor rep10\n",
      "           gsm         title\n",
      "0   GSM5574685    tumor rep1\n",
      "1   GSM5574686    tumor rep2\n",
      "2   GSM5574687    tumor rep3\n",
      "3   GSM5574688    tumor rep4\n",
      "4   GSM5574689    tumor rep5\n",
      "5   GSM5574690    tumor rep6\n",
      "6   GSM5574691    tumor rep7\n",
      "7   GSM5574692    tumor rep8\n",
      "8   GSM5574693    tumor rep9\n",
      "9   GSM5574694   tumor rep10\n",
      "10  GSM5574695   tumor rep11\n",
      "11  GSM5574696   tumor rep12\n",
      "12  GSM5574697   tumor rep13\n",
      "13  GSM5574698   tumor rep14\n",
      "14  GSM5574699   tumor rep15\n",
      "15  GSM5574700   tumor rep16\n",
      "16  GSM5574701   tumor rep17\n",
      "17  GSM5574702   tumor rep18\n",
      "18  GSM5574703   tumor rep19\n",
      "19  GSM5574704   tumor rep20\n",
      "20  GSM5574705   tumor rep21\n",
      "21  GSM5574706   tumor rep22\n",
      "22  GSM5574707   tumor rep23\n",
      "23  GSM5574708   tumor rep24\n",
      "24  GSM5574709   tumor rep25\n",
      "25  GSM5574710   tumor rep26\n",
      "26  GSM5574711   tumor rep27\n",
      "27  GSM5574712   tumor rep28\n",
      "28  GSM5574713   tumor rep29\n",
      "29  GSM5574714   tumor rep30\n",
      "30  GSM5574715   normal rep1\n",
      "31  GSM5574716   normal rep2\n",
      "32  GSM5574717   normal rep3\n",
      "33  GSM5574718   normal rep4\n",
      "34  GSM5574719   normal rep5\n",
      "35  GSM5574720   normal rep6\n",
      "36  GSM5574721   normal rep7\n",
      "37  GSM5574722   normal rep8\n",
      "38  GSM5574723   normal rep9\n",
      "39  GSM5574724  normal rep10\n",
      "40  GSM5574725  normal rep11\n",
      "41  GSM5574726  normal rep12\n",
      "42  GSM5574727  normal rep13\n",
      "43  GSM5574728  normal rep14\n",
      "44  GSM5574729  normal rep15\n",
      "45  GSM5574730  normal rep16\n",
      "46  GSM5574731  normal rep17\n",
      "47  GSM5574732  normal rep18\n",
      "48  GSM5574733  normal rep19\n",
      "49  GSM5574734  normal rep20\n",
      "50  GSM5574735  normal rep21\n",
      "51  GSM5574736  normal rep22\n",
      "52  GSM5574737  normal rep23\n",
      "53  GSM5574738  normal rep24\n",
      "54  GSM5574739  normal rep25\n",
      "55  GSM5574740  normal rep26\n",
      "56  GSM5574741  normal rep27\n",
      "57  GSM5574742  normal rep28\n",
      "58  GSM5574743  normal rep29\n",
      "59  GSM5574744  normal rep30\n"
     ]
    }
   ],
   "source": [
    "import GEOparse\n",
    "import pandas as pd\n",
    "\n",
    "# Load GEO metadata\n",
    "gse = GEOparse.get_GEO(\"GSE183947\", destdir=\"./data\")\n",
    "\n",
    "meta = []\n",
    "\n",
    "for gsm_name, gsm in gse.gsms.items():\n",
    "    title = gsm.metadata.get(\"title\", [\"\"])[0]  # sample title\n",
    "    meta.append({\n",
    "        \"gsm\": gsm_name,\n",
    "        \"title\": title\n",
    "    })\n",
    "\n",
    "# Create DataFrame\n",
    "pheno = pd.DataFrame(meta)\n",
    "print(\"Pheno shape:\", pheno.shape)\n",
    "print(pheno.head(10))\n",
    "print(pheno)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 56,
   "id": "8ca672d2",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Index(['Unnamed: 1', 'Unnamed: 2', 'Unnamed: 3', 'Unnamed: 4', 'Unnamed: 5',\n",
      "       'Unnamed: 6', 'Unnamed: 7'],\n",
      "      dtype='object')\n",
      "Index(['gsm', 'title'], dtype='object')\n",
      "          gsm       title\n",
      "0  GSM5574685  tumor rep1\n",
      "1  GSM5574686  tumor rep2\n",
      "2  GSM5574687  tumor rep3\n",
      "3  GSM5574688  tumor rep4\n",
      "4  GSM5574689  tumor rep5\n"
     ]
    }
   ],
   "source": [
    "print(df.columns[ 1:10])\n",
    "print(pheno.columns)\n",
    "print(pheno.head(5))\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "cf473bea",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Look for GSM IDs in column names\n",
    "for col in df.columns[:10]:\n",
    "    matches = [gsm for gsm in pheno['gsm'] if gsm in col]\n",
    "    if matches:\n",
    "        print(col, matches)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "9dcd32e0",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Example mapping (must match your experiment)\n",
    "sample_map = {\n",
    "    'CA.102548': {'condition': 'tumor', 'replicate': 1},\n",
    "    'CA.104338': {'condition': 'tumor', 'replicate': 2},\n",
    "    'CA.105094': {'condition': 'tumor', 'replicate': 3},\n",
    "    # ... continue for all 60 samples\n",
    "}\n",
    "\n",
    "# Convert to DataFrame\n",
    "pheno_new = pd.DataFrame.from_dict(sample_map, orient='index')\n",
    "pheno_new.index.name = 'sample_id'\n",
    "pheno_new.reset_index(inplace=True)\n",
    "\n",
    "# Align expression data\n",
    "df = df[pheno_new['sample_id']]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "2e234b9a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Expression matrix shape: (20246, 3)\n",
      "Metadata shape: (3, 3)\n",
      "Index(['CA.102548', 'CA.104338', 'CA.105094'], dtype='object')\n",
      "   sample_id condition  replicate\n",
      "0  CA.102548     tumor          1\n",
      "1  CA.104338     tumor          2\n",
      "2  CA.105094     tumor          3\n"
     ]
    }
   ],
   "source": [
    "print(\"Expression matrix shape:\", df.shape)\n",
    "print(\"Metadata shape:\", pheno_new.shape)\n",
    "print(df.columns[:5])\n",
    "print(pheno_new.head())\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "id": "5e766939",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Expression matrix shape: (20246, 60)\n",
      "First 10 columns: Index(['CA.102548', 'CA.104338', 'CA.105094', 'CA.109745', 'CA.1906415',\n",
      "       'CA.1912627', 'CA.1924346', 'CA.1926760', 'CA.1927842', 'CA.1933414'],\n",
      "      dtype='object')\n",
      "Metadata shape: (60, 3)\n",
      "Expression matrix aligned shape: (20246, 60)\n"
     ]
    }
   ],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import requests, gzip\n",
    "from io import BytesIO\n",
    "\n",
    "# Download FPKM CSV\n",
    "url = \"https://ftp.ncbi.nlm.nih.gov/geo/series/GSE183nnn/GSE183947/suppl/GSE183947_fpkm.csv.gz\"\n",
    "r = requests.get(url)\n",
    "with gzip.GzipFile(fileobj=BytesIO(r.content)) as f:\n",
    "    expr_df = pd.read_csv(f, index_col=0)\n",
    "\n",
    "print(\"Expression matrix shape:\", expr_df.shape)\n",
    "print(\"First 10 columns:\", expr_df.columns[:10])\n",
    "\n",
    "# Step 1: Create metadata based on column order\n",
    "sample_ids = expr_df.columns.tolist()\n",
    "num_samples = len(sample_ids)\n",
    "\n",
    "# Example: 60 samples, first 30 = tumor, next 30 = normal, 5 replicates each (adjust as needed)\n",
    "conditions = ['tumor']*30 + ['normal']*30\n",
    "replicates = list(range(1,6))*12  # 12 sets of 5 replicates\n",
    "\n",
    "metadata = pd.DataFrame({\n",
    "    'sample_id': sample_ids,\n",
    "    'condition': conditions,\n",
    "    'replicate': replicates\n",
    "})\n",
    "\n",
    "# Step 2: Expression matrix columns match metadata\n",
    "expr_df = expr_df[metadata['sample_id']]\n",
    "\n",
    "print(\"Metadata shape:\", metadata.shape)\n",
    "print(\"Expression matrix aligned shape:\", expr_df.shape)\n",
    "\n",
    "# Optional: Save CSV\n",
    "expr_df.to_csv(\"GEO/expression_matrix.csv\")\n",
    "metadata.to_csv(\"GEO/sample_metadata.csv\", index=False)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "id": "75b05875",
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "expr_log = np.log2(expr_df + 1)\n",
    "\n",
    "expr_log.to_csv(\"GEO/GSE183947_fpkm_log2.csv\")\n",
    "metadata.to_csv(\"GEO/GSE183947_metadata.csv\", index=False)\n",
    "\n"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "myytcode",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
